Lib/test/libregrtest/run_workers.py

```diff
@@ -394,15 +402,15 @@ def run(self) -> None:
                 except StopIteration:
                     break
 
-                self.start_time = time.monotonic()
+                self.start_time = start_time = time.monotonic()
```

Comment:
To avoid the local `start_time` variable, you can initialize self.start_time to time.monotonic() in the constructor (`__init__()`).